x-pack/filebeat/input/netflow/testdata/golden/IPFIX-Barracuda-extended-uniflow-template-256.golden.json

```diff
@@ -0,0 +1,153 @@
+{
+  "test_name": "IPFIX Barracuda extended uniflow template 256",
+  "events": [
+    {
+      "Timestamp": "2018-04-18T08:16:47Z",
+      "Meta": null,
+      "Fields": {
+        "destination": {
+          "ip": "64.235.151.76",
+          "locality": "public",
+          "port": 443
+        },
+        "device": {
+          "ip": "192.0.2.1"
+        },
+        "event": {
+          "action": "netflow",
+          "created": "2018-04-18T08:16:47Z",
+          "duration": 0
+        },
+        "flow": {
+          "id": "xrwrnNR_Xgs",
+          "locality": "private"
+        },
+        "netflow": {
+          "AuditCounter": 4157725,
+          "BindIPv4Address": "213.208.150.99",
+          "BindTransportPort": 64238,
+          "ConnIPv4Address": "64.235.151.76",
+          "ConnTransportPort": 443,
+          "FW_Rule": "MTH:MTH-MC-to-Inet",
+          "LogOp": 1,
+          "Reason": 0,
+          "ReasonText": "Normal Operation",
+          "ServiceName": "https",
+          "Timestamp": 1524039407,
+          "TrafficType": 0,
+          "destinationIPv4Address": "64.235.151.76",
+          "destinationTransportPort": 443,
+          "egressInterface": 3689,
+          "exporter": {
+            "address": "192.0.2.1:4444",
+            "sourceId": 0,
+            "timestamp": "2018-04-18T08:16:47Z",
+            "uptimeMillis": 0,
+            "version": 10
+          },
+          "firewallEvent": 1,
+          "flowDurationMilliseconds": 0,
+          "flowEndSysUpTime": 1957197969,
+          "flowStartSysUpTime": 1957197969,
+          "ingressInterface": 35233,
+          "octetDeltaCount": 0,
+          "octetTotalCount": 0,
+          "packetDeltaCount": 0,
+          "packetTotalCount": 0,
+          "protocolIdentifier": 6,
```

Comment:
I assume this proto identifier correspond to the IANA number? If so, put it in `network.iana_number`

Reply:
Will do.